### ISSUE TITLE
ERROR: Unable to load Spoon: SpoonInstall

### ISSUE THREAD

**Issue by tim0120** [dbalatero/VimMode.spoon#100]
Every time that I reload the config (i.e., open the application or reload the config manually), I get the title error message in the Hammerspoon console:
```*** ERROR: Unable to load Spoon: SpoonInstall```
I also get the following error notification (pictured):
<img width="351" alt="Screenshot 2022-11-15 at 11 23 29" src="https://user-images.githubusercontent.com/39891386/201972234-a34476b2-05c7-4988-ba37-00e602f2eed8.png">
Is there any fix to this (e.g., fixing something in the `init.lua` (not edited btw))?
Thanks for taking a look!

**Comment by levabala** [comment]
Got the same thing 
For me it was incorrect initial install - wget was missing
Then run install script again and it works

**Comment by adisidev** [comment]
Yep, same! `brew install wget` fixes everything (presuming you already have homebrew).
